README.md

```diff
@@ -149,17 +157,6 @@ greater than 2.6.
 
 * Pep8 (style analysis).
 
-## Running from source
-
-It is possible to run Spyder directly (i.e. without installation)
-from the unpacked zip folder (see *Installing from source*) using
-Spyder's bootstrap script like this:
```

Comment:
unpacked zip folder or git repository!
Also, having a reference to "Installing from source" right after saying that you don't need to install it is a bit disturbing, but maybe it's only me.